src/cloud.js

```diff
@@ -47,8 +47,8 @@ function Cloud() {
 }
 
 Cloud.prototype.init = function () {
-    this.urlBasePath = '/api/v1';
-    this.url = this.determineCloudDomain() + this.urlBasePath;
+    this.apiBasePath = '/api/v1';
```

Comment:
Not strictly necessary, but I think it's helpful to differentiate the API routes from other pages.